apps/src/aichat/views/ChatEventView.tsx

```diff
@@ -48,41 +49,52 @@ function formatModelUpdateText(update: ModelUpdate): string {
  */
 const ChatEventView: React.FunctionComponent<ChatEventViewProps> = ({
   event,
+  isTeacherView,
 }) => {
   const dispatch = useAppDispatch();
 
   if (isChatMessage(event)) {
-    return <ChatMessage {...event} />;
```

Comment:
nit: can this just be `<ChatMessage {...event} isTeacherView={isTeacherView}/>`?